eventmesh-connectors/eventmesh-connector-http/src/main/java/org/apache/eventmesh/connector/http/sink/config/HttpRetryConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.connector.http.sink.config;
+
+import lombok.Data;
+
+@Data
+public class HttpRetryConfig {
+    // maximum number of attempts to retry, default 3, if set to 0 or 1, no retry
+    private int maxAttempts = 3;
```

Comment:
As expressed by `maxAttempts`, it signifies the maximum number of attempts rather than the number of retries after the initial call fails. Strictly speaking, the minimum valid value for this field is 1 because we must invoke the `handle` method at least once; values less than 1 are illegal. Do we need to strictly define the range of acceptable values for this field, or should we overlook illegal values like 0?

Reply:
Since this config is under the `retryConfig`, so I think it would be less misleading to make `maxAttempts` uniformly minus 1 and make it be `retryNums`.

Reply:
If an unexcepted value is received, you can throw an exception and fail the startup.

Reply:
This is a good suggestion. I will rename 'maxAttempts' to 'maxRetries' and validate it when reading the configuration information.

Reply:
Resolved at https://github.com/apache/eventmesh/pull/4837/commits/9c9e4a85706e42e5da00381ac4447644ca6f4d14